src/main/webapp/css/navigation-bar.css

```diff
@@ -1,11 +1,12 @@
-/* Remove bullet points from navigation bar. */
-nav ul {
-  list-style-type: none;
-  padding-left: 0;
+nav.navBarTransparent {
+  background-color: rgba(255, 255, 255, 0) !important;
+  border-bottom: 1px solid rgba(207, 203, 203, 0.4);
 }
 
-/* Keep navigation menu on one line. */
-nav li {
-  display: inline-block;
-  margin-right: 10px
+nav.navbar {
+  /* background-color: rgba(95, 103, 112, 1); */
```

Comment:
Was this supposed to uncommented? Just confirming

Reply:
Oh yes supposed to be deleted.